src/ALCProxy.Communication/ICommObjects.cs

```diff
@@ -0,0 +1,257 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. 
+// The .NET Foundation licenses this file to you under the MIT license. 
+// See the LICENSE file in the project root for more information. 
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Loader;
+
+namespace ALCProxy.Communication
```

Comment:
I wasn't exactly what to name the file at the time, since it holds both the code for servers and clients and they both fall under the section of the proxies that handles communication between ALCs. Would it make more sense to split up those two into separate files and rename the files to match with each?